manifest.json

```diff
@@ -309,5 +309,10 @@
         "update.js": "bdc68c213786748d597b1800bcae733c294161d5e0797adbcfb364ebe5240732",
         "webui/index.html": "fe1e877ad7805a1feb6a97bac5f430a2b3b8eca8c014d8d80224652f15ede635",
         "webui/js/main.js": "e556a0586af38ed1d14e57ff3ce734e0866766ef6d8b2cb178baddd1bd5e063f"
+        "files/entities/misc/effect_everyone_loves_larpa.xml": "5d7bb7afe566dfd35a3c23fada2fe254748fb35caa046cbb0f5ece1b48f3a385",
+        "files/scripts/effect_everyone_loves_larpa.lua": "9c2b813576603c5aef563417f7de54ab4eaef57242d9244981159efd41532416",
+        "files/scripts/effect_everyone_loves_larpa_end.lua": "e105887a9f97c77252af638361b811a59ed24d27f77cb127637935ed9773ff3c",
+        "files/ui_gfx/status_indicators/everyone_loves_larpa_ui.png": "5455d320ac7667ffe4209a3da64230bea0fbfb8fb16dc03adb79b8b701b89d2c",
+        "twitch_fragments/outcomes/everyone_loves_larpa.lua": "363a18c3a23412b5047b8e5431f3e41737d9162960fd532224d725b5fdf1152d"

```

Comment:
Could you move those around to maintain alphabetic order in the file? Beside esthetic code (my main reason) it could potentially help with merge conflicts - multiple PR could potentially try to add in the same spot (after last row and before brackets), so manual intervention could be needed.